pydis_site/apps/api/viewsets/bot/aoc_link.py

```diff
@@ -0,0 +1,71 @@
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework.mixins import (
+    CreateModelMixin, DestroyModelMixin, ListModelMixin, RetrieveModelMixin
+)
+from rest_framework.viewsets import GenericViewSet
+
+from pydis_site.apps.api.models.bot import AocAccountLink
+from pydis_site.apps.api.serializers import AocAccountLinkSerializer
+
+
+class AocAccountLinkViewSet(
+    GenericViewSet, CreateModelMixin, DestroyModelMixin, RetrieveModelMixin, ListModelMixin
+):
+    """
+    View providing management for Users who linked their AoC accounts to their Discord Account.
+
+    ## Routes
+
+    ### GET /bot/aoc-account-links
+    Returns all the AoC account links
+
+    #### Response format
+    >>> [
+    ...     {
+    ...         "user": 2,
+    ...         "aoc_username": "AoCUser1"
+    ...     },
+    ...     ...
+    ... ]
+
+
+    ### GET /bot/aoc-account-links<user__id:int>
+    Retrieve a AoC account link by User ID
+
+    #### Response format
+    >>> [
+    ...     {
+    ...         "user": 2,
+    ...         "aoc_username": "AoCUser1"
+    ...     },
+    ... ]
```

Comment:
Does this really still return a list for just 1 user? I am pretty sure the default DRF behaviour is to return a single object.

Reply:
Yes, I was under the impression that it does, but after testing it out yesterday, it seemed to return a list still.

Reply:
Don't mind me, you're right! :D 
Not sure why I updated in the end.